lib/AST/ASTImporter.cpp

```diff
@@ -6084,11 +6076,11 @@ ExpectedStmt ASTNodeImporter::VisitVAArgExpr(VAArgExpr *E) {
 
 
 ExpectedStmt ASTNodeImporter::VisitGNUNullExpr(GNUNullExpr *E) {
-  ExpectedType TypeOrErr = Importer.Import(E->getType());
+  ExpectedType TypeOrErr = import(E->getType());
   if (!TypeOrErr)
     return TypeOrErr.takeError();
 
-  ExpectedSLoc LocStartOrErr = Importer.Import(E->getLocStart());
+  ExpectedSLoc LocStartOrErr = import(E->getLocStart());
```

Comment:
What is the difference between `Importer.Import` and `import`? Is it just syntactic sugar?